yorkie/src/main/kotlin/dev/yorkie/document/Document.kt

```diff
@@ -395,16 +391,17 @@ public class Document(
      * Triggers an event in this [Document].
      */
     private suspend fun publishPresenceEvent(presences: Presences) {
-        val iterator = presenceEventQueue.listIterator()
+        val publishedEvents = mutableListOf<Event.PresenceChange>()
+        val iterator = presenceEventQueue.toList().iterator()
```

Comment:
since it is changed to copying the list, we could just use forEach for the loop

Reply:
done in [8643e03](https://github.com/yorkie-team/yorkie-android-sdk/pull/163/commits/8643e037748755c89d3190be04cf72bbb02f66c7)